contracts/LifToken.sol

```diff
@@ -1,472 +1,17 @@
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.13;
 
 import "zeppelin-solidity/contracts/ownership/Ownable.sol";
-import "zeppelin-solidity/contracts/payment/PullPayment.sol";
-import 'zeppelin-solidity/contracts/math/SafeMath.sol';
-import './LifInterface.sol';
-import './LifDAOInterface.sol';
+import "zeppelin-solidity/contracts/token/MintableToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/lifecycle/Pausable.sol";
```

Comment:
do we have to import the `SafeMath` and `Ownable` too? i don't see it used in the code.